_sass/testimonials.scss

```
@@ -0,0 +1,13 @@
+.testimonial-row {
+  &:nth-of-type(even) {
+    flex-direction: row-reverse;
+  }
+}
+
+.testomonial-avatar {
+  border-radius: 50%;
+  width: 15rem;
+  @include media-query("medium") {
+    width: 100%;
+  }
+}
```

Might want to organise it like this instead

```suggestion
.testimonial-row {
  &-row {
    &:nth-of-type(even) {
      flex-direction: row-reverse;
    }
  }
  
  &-avatar {
    border-radius: 50%;
    width: 15rem;
    @include media-query("medium") {
      width: 100%;
    }
  }
}
```